priv/repo/migrations/20200601193405_create_glific_tables.exs

```diff
@@ -107,9 +107,11 @@ defmodule Glific.Repo.Migrations.GlificCore do
       # jsonb object of out_of_office data which is a bit convoluted to represent as columns
       add :out_of_office, :jsonb
 
-      # organization can be
+      # organization services can be changed to inactive
       add :is_active, :boolean, default: true
 
+      add :timezone, :string, default: "Asia/Kolkata"
```

Comment:
I think we should not have this as default in the migration file. Currently we are duplicating the default value. 
Do you think since this is not going to use so much we should remove it from here?